xla/stream_executor/cuda/cudnn_frontend_helpers.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef XLA_STREAM_EXECUTOR_CUDA_CUDNN_FRONTEND_HELPERS_H_
+#define XLA_STREAM_EXECUTOR_CUDA_CUDNN_FRONTEND_HELPERS_H_
+
+namespace stream_executor {
+namespace gpu {
+
+#define RETURN_IF_CUDNN_FRONTEND_ERROR(expr)                                \
```

Comment:
`#define RETURN_IF_CUDNN_FRONTEND_ERROR(expr)` appears in cuda_dnn.cc:131 and my compiles choke on this.

Reply:
that's a small race with https://github.com/openxla/xla/commit/09ca1ed7aa37e3555e486a32e9b4cf0df3f47bd8